python/ray/train/tests/test_tune.py

```diff
@@ -302,7 +302,6 @@ def test_run_config_in_trainer_and_tuner(
         assert not (tmp_path / "trainer").exists()
         assert both_msg not in caplog.text
     else:
-        assert tuner._local_tuner.get_run_config() == RunConfig()
```

Comment:
Is it because we inject a default storage context into the run config if not specified?